src/shared/organisms/DataPanel/DataPanel.tsx

```diff
@@ -99,8 +98,19 @@ function findIndexes(arr: any[], predicate: any) {
   });
   return indexes;
 }
-function removeIndeces(arr: any[], indicesToRemove: number[]) {
-  return arr.filter((_, index) => !indicesToRemove.includes(index));
+
+function getPathForChildResource(resource: any, parent: ResourceObscured) {
+  const extension = resource._filename?.split('.').pop() ?? 'txt';
```

Comment:
same here, are we sure that the data type is always `txt` in studios

Reply:
got it. removed.